packages/hash/subgraph/src/types/element.ts

```diff
@@ -46,20 +46,22 @@ export type PropertyObject = {
   [_: BaseUri]: PropertyValue;
 };
 
+export type EntityMetadata = {
```

Comment:
Thanks for doing this, @benwerner01 had a use for it in his PR too. We could of course always just do `Entity["metadata"]` but I think this is probably nicer 👍